src/test/scala/zio/prelude/CoherentSpec.scala

```diff
@@ -16,15 +17,17 @@ object CoherentSpec extends DefaultRunnableSpec {
       },
       test("IdentityEqual") {
         val instance = implicitly[EqualIdentity[String]]
-
         assert(instance.identity)(isEmptyString) &&
         assert(instance.combine("a", "b"))(equalTo("ab")) &&
+        assert(instance.repeat("a", 2))(equalTo("aa")) &&
```

Comment:
Instead of 2, could we test with a higher number, like 5, so that we defend against the bug we've encountered earlier?